nservicebus/sqlserver/configuration.md

```diff
@@ -165,7 +106,7 @@ snippet:sqlserver-TimeToWaitBeforeTriggeringCircuitBreaker
 
 ### Pause Time
 
-Overrides the default time to pause after a failure while trying to receive a message.
+Overrides the default time to pause after a failure while trying to receive a message. The setting has been made obsolete in version 3.
```

Comment:
Maybe it is better to state that this setting is only available in version 2 and earlier?